Yona/Yona/SignUp/SignUpFirstStepViewController.swift

```diff
@@ -151,13 +151,8 @@ extension SignUpFirstStepViewController: UITextFieldDelegate {
     }
     
     func nextTextField() {
-        if lastnameTextField.isFirstResponder() {
-            firstnameTextField.resignFirstResponder()
-            lastnameTextField.resignFirstResponder()
-        } else {
-            firstnameTextField.resignFirstResponder()
-            lastnameTextField.becomeFirstResponder()
-        }
+        firstnameTextField.resignFirstResponder()
+        lastnameTextField.isFirstResponder() ? lastnameTextField.resignFirstResponder() : lastnameTextField.becomeFirstResponder()
```

Comment:
I don't like repletion of code in if statement. In this case `firstnameTextField.resignFirstResponder()` is called always and then the `?:` is more useful in this case instead when the result of the check is only on instruction.